index.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-restricted-syntax */
+
+import { addBook, removeBook, populateBooksFromStorage } from './modules/books-manager.js';
+import { $select, $selectById } from './modules/selectors.js';
+
+const anchors = $select('.navbar a', true);
+
+function switchSection(event) {
+  event.preventDefault();
+
+  this.classList.toggle('active', true);
+
+  let recentSectionId;
```

Comment:
- [ ] Same here, Please use arrow function syntax here instead of regular function syntax because in this project you learn how to use  ES6 new features.

Reply:
As an arrow function doesn't provide reference to the element on which the event occurred, therefore, a regular function is the best bet in this scenario.